exchanges/binance/binance_wrapper.go

```diff
@@ -956,10 +956,15 @@ func (b *Binance) SubmitOrder(ctx context.Context, s *order.Submit) (order.Submi
 			return submitOrderResponse, fmt.Errorf("invalid side")
 		}
 
-		var oType string
+		var (
+			oType       string
+			timeInForce string
+		)
+
 		switch s.Type {
 		case order.Limit:
 			oType = cfuturesLimit
+			timeInForce = "GCT"
```

Comment:
Eyeballing this seems like it needs to change from GCT -> GTC

Reply:
wow, that's a Freudian slip right there

Reply:
🤣